MAINTAINERS.md

```diff
@@ -1,3 +1,26 @@
+## Specification maintainers
+
+### SAM/BAM
+
+* James Bonfield (@jkbonfield)
+* Yossi Farjoun (@yfarjoun)
+
+### CRAM
+
+* James Bonfield (@jkbonfield)
+* Vadim Zalunin (@vadimzalunin)
```

Comment:
Indeed he is, we have a transition plan at EBI via @AlexanderSenf and @AlexanderVi
For now, it is fine to list Vadim.